### ISSUE TITLE
Please someone fork this and revive it

### ISSUE THREAD

**Issue by propagare** [JulianSchuette/tiddlysnip#6]
It is such an essential tool, I can't believe it was abandoned because it doesn't get enough money/applause. Such a wast.... :-/

**Comment by JulianSchuette** [comment]
I agree. I am happy to hand over this project to whomever finds the time to resurrect it!
